### PR TITLE
StyleCI / PHP-CS-Fixer

### PR DESCRIPTION
**Pull request by Soullivaneuh**
This bridge permits developers to fix StyleCI issues on command line with php-cs-fixer tool or check it before make a PR.

### CODE REVIEW

File: .styleci.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+preset: symfony
```
Comment (sstok) [inline .styleci.yml]: Can you add the following?:

```yaml
finder:
    path:
        - 'src/'
        - 'tests/'
```

Comment (Soullivaneuh) [inline .styleci.yml]: Done.

**Comment by sstok** [comment]
Thanks you @Soullivaneuh

**Comment by Soullivaneuh** [comment]
Thank you too!

**Pull request merged**